package inheritance.week7_assignment;

public class Painting extends ArtPiece {

    private String title;

    public Painting(String title) {
        super();
        this.title = title;
    }

    @Override
    public String describe() {

        return "Painting: "
                + title
                + ", framed on canvas";
    }
}